docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[search-using-query-rules]]
+== Searching with query rules
+++++
+<titleabbrev>Searching with query rules</titleabbrev>
+++++
+
+[[query-rules]]
+
+preview::[]
+
+_Query rules_ are rules that are defined inside {es} and are applied only to queries that match these rules.
+This allows more control over promoted results for specific queries only. For example, you may want to promote results only when users are searching for specific items.
+
+Rules are defined using the <<query-rules-apis,Query rules API>>.
+When defining a rule, we take into consideration the following:
```

Comment:
I'm not sure who "we" is in this sentence and list. Author and reader? Elasticsearch?

If author/reader, I'd revise to avoid "we". For example: "When defining a rule, consider the following:".

If Elasticsearch, replace "we" with "Elasticsearch".